dist/css/bootstrap.css

```diff
@@ -3987,7 +3987,7 @@ input[type="button"].btn-block {
 
 .card-header {
   padding: 0.75rem 1.25rem;
-  background-color: #f5f5f5;
+  background-color: whitesmoke;
```

Comment:
I'll close this and give it another go. It might had been a version sass that added colournames, instead of hex.